pypi-release.yml

```diff
@@ -0,0 +1,34 @@
+# File: simple-param.yml
+parameters:
+- name: repoName
+  type: string
+- name: wDirectory
+  type: string
+
+steps:
+  # Compare latest tag and a tag previous to that tag
+  - bash: |
+      latest_tag=`git tag --sort=-creatordate | sed -n '1p'`
+      previous_tag=`git tag --sort=-creatordate | sed -n '2p'`
+      actual_diff=`git diff $latest_tag $previous_tag`
+      if [ -z "$actual_diff" ]; then
+        echo There is no difference between last two commits namely $latest_tag and $previous_tag. Aborting release!!! >&2
+      fi
+    failOnStderr: true
+    displayName: "Check for changes in this release"
```

Comment:
What motivates checking for differences between release tags? I would have assumed a human made the new release tag to trigger a release to pypi. Why stop such a human when the release they tagged is identical to the previous one? Does it hurt anything to do a release with a new version number that's identical to a previous release?

Reply:
@Patater To ensure that same package is not released multiple times. The trigger behind the release automation is still the tag pushed by a human, but this step ensures *accidental* tags are not released to pypi. Why would one wants to release same package multiple times?

Reply:
Seems odd to try to block this. If nothing breaks, and sombody makes and pushes a duplicate content tag on accident, one would expect a duplicate content release on pypi as well. Less code to maintain if we don't try to block stupidity and focus purely on mechanism and not policy.

Reply:
@Patater OK. I'll remove `Check for changes in this release`